src/test/skript/tests/syntaxes/conditions/CondTooltip.sk

```diff
@@ -0,0 +1,15 @@
+test "tooltip" when running minecraft "1.20.5":
+
+	set {_item} to diamond
```

Comment:
Avoid potential incorrect parse. Using `a` forces the parse to be a ClassInfo lookup aka an ItemType.
```suggestion
	set {_item} to a diamond
```